ssh.go

```diff
@@ -72,13 +72,16 @@ type ConnectionFailedCallback func(conn net.Conn, err error)
 type Window struct {
 	Width  int
 	Height int
+
+	WidthPixels  int
```

Comment:
Document these while you're here, clarifying that for all of these fields, zero means unspecified, and when all are given, that Width/Height (which you should now clarify are rows/columns) take precedence. Maybe link to the spec.

Reply:
Done